frontend/src/store/modules/shoots/index.js

```diff
@@ -12,33 +12,23 @@ import omit from 'lodash/omit'
 import map from 'lodash/map'
 import get from 'lodash/get'
 import replace from 'lodash/replace'
-import transform from 'lodash/transform'
-import isEqual from 'lodash/isEqual'
-import isObject from 'lodash/isObject'
-import orderBy from 'lodash/orderBy'
-import toLower from 'lodash/toLower'
-import padStart from 'lodash/padStart'
 import filter from 'lodash/filter'
 import includes from 'lodash/includes'
 import some from 'lodash/some'
-import split from 'lodash/split'
-import join from 'lodash/join'
 import set from 'lodash/set'
 import head from 'lodash/head'
 import sample from 'lodash/sample'
 import isEmpty from 'lodash/isEmpty'
 import cloneDeep from 'lodash/cloneDeep'
-import semver from 'semver'
-import store from '../'
+import store from '../../'
```

Comment:
brauchen wir den store hier noch? Können wir das nicht auch gleich los werden?

Reply:
Ok. We still need the store in the `HANDLE_EVENTS` mutation. We should not remove it in this PR.